md/neumorphism-login-page.md

```diff
@@ -0,0 +1,127 @@
+# Neumorphosm Login page with Bonita Community edition
+
+In the late 2019, a new User Interface design style called Neumorphis became popular and people believe that it will become a biggest trend in 2020.
```

Comment:
```suggestion
In the late 2019, a new User Interface design style called Neumorphism became popular and people believe that it will become a biggest trend in 2020.
```